### PR TITLE
Micro app - sceneview viewpoint navigation

### PR DESCRIPTION
**Pull request by puneet-pdx**

### CODE REVIEW

File: microapps/SceneViewSetViewpointApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewsetviewpointapp/screens/MainScreen.kt [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    val catalinaCamera = remember { Camera(catalina, 122.5, 77.0, 0.0) }
+    val goldenGateBridgeCamera = remember { Camera(goldenGateBridge, 111.3, 71.7, 0.0) }
+
+    val scale = remember { 170000.0 }
+    val bookmark = remember {
+        Bookmark(
+            "disneyLand",
+            Viewpoint(disneyLand, scale)
+        )
+    }
```
Comment (hud10837) [inline microapps/SceneViewSetViewpointApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewsetviewpointapp/screens/MainScreen.kt:146]: Should we have a unique place here so they're all different? I recommend Edinburgh - the castle is at 55.9486° N, 3.1999° W

File: microapps/SceneViewSetViewpointApp/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# SceneView Set Viewpoint Micro-app
+
+This micro-app demonstrates the use of `SceneViewpointOperation` with a composable `SceneView`.
+
+![Screenshot](screenshot.png)
```
Comment (hud10837) [inline microapps/SceneViewSetViewpointApp/README.md:5]: The screenshot is very large, can you do it like [floor filter](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/tree/main/toolkit/indoors#floorfilter)?

```suggestion
|SceneView Set Viewpoint Micro-app|
|:--:|
|![image](screenshot.png)|
```

**Review by hud10837 — APPROVED** [review]
LGTM, two small changes but no need to re-review

File: microapps/SceneViewSetViewpointApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewsetviewpointapp/screens/MainScreen.kt [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+ */
+
+package com.arcgismaps.toolkit.sceneviewsetviewpointapp.screens
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.MoreVert
+import androidx.compose.material3.CircularProgressIndicator
```
Comment (changanxian) [inline microapps/SceneViewSetViewpointApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewsetviewpointapp/screens/MainScreen.kt:26]: unused import

File: microapps/SceneViewSetViewpointApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewsetviewpointapp/screens/MainScreen.kt [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
```
Comment (changanxian) [inline microapps/SceneViewSetViewpointApp/app/src/main/java/com/arcgismaps/toolkit/sceneviewsetviewpointapp/screens/MainScreen.kt:38]: unused import

**Review by changanxian — APPROVED** [review]
LGTM

File: microapps/SceneViewSetViewpointApp/README.md (hunk 2 above)
Comment (puneet-pdx) [reply microapps/SceneViewSetViewpointApp/README.md]: updated

**Pull request merged**